static/docker-compose.yml

```diff
@@ -184,6 +184,7 @@ services:
       REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
       REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
       serviceConfig: ${serviceConfig:-}
```

Comment:
env var lines up with config 👍 

did we manage to add some alerting around the 401? That seems like it should scream at us.

Reply:
👍 Great callout. Looking into that separately.